### PR TITLE
More pipe changes

### PR DESCRIPTION
**Pull request by Readystorm546**
# General Documentation

### Intent of your Pull Request

I fix some more piping in one commit 

### Briefly describe your PR and the impacts of it, in layman's terms. 

fixes some piping.

### What general grouping does this PR fall under? 

Mapping

:cl:  
tweak: tweaked a few pipes to make them not shit
/:cl:

### CODE REVIEW

**Review by patpol4 — CHANGES_REQUESTED** [review]
No reason to change this disposals pipe really, it's not as bad as the other one.
Also you have step vars in your pr again

**Comment by Readystorm546** [comment]
> No reason to change this disposals pipe really, it's not as bad as the other one.
> Also you have step vars in your pr again

its pretty bad but whatever. ill accept it
